src/Spryker/Zed/EventBehavior/Persistence/Propel/Behavior/EventBehavior.php

```diff
@@ -435,10 +443,24 @@ protected function addSaveEventBehaviorEntityChangeMethod()
  */
 protected function saveEventBehaviorEntityChange(array \$data)
 {
+    \$encodedData = json_encode(\$data);
+    \$dataLength = strlen(\$encodedData);
+
+    if (\$dataLength > $maxRecommendedEventMessageDataSize * 1024) {
+        \$warningMessage = sprintf(
+            '%s event message data size (%d KB) exceeds the allowable limit of %d KB.',
```

Comment:
please add suggestion for action, like please reduce event message size or it might disrupt P&S process